Source/Core/VideoBackends/OGL/Render.cpp

```diff
@@ -1241,7 +1245,7 @@ void Renderer::SetViewport()
 		};
 		glViewport(iceilf(X), iceilf(Y), iceilf(Width), iceilf(Height));
 	}
-	glDepthRangef(GLNear, GLFar);
+	glDepthRangef(GLFar, GLNear);
```

Comment:
Just curious, what does fifoci says if we normalize this value by dividing with double(0xffffff) and using the double version of this call?